src/sections/Home/components/AnimatedRole/HomeRole.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { makeStyles, Typography } from "@material-ui/core";
+import { styles } from "../../../../styles/styles";
```

Comment:
I think you can start with "/" instead of "./" and this would make the url shorter